CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # did-veres-one ChangeLog
 
+## 7.0.0 - 2019-08-01
```

Comment:
Leave date "TBD" or somesuch.  For the release flow, it is the hope that the person doing the release *just* has to set this date properly and then run the release script.  The changelog *must* have unstaged changes for the release script to work.

Reply:
Np, changed to TBD.